src/controllers/dlgprefcontroller.cpp

```diff
@@ -21,6 +21,17 @@
 #include "preferences/usersettings.h"
 #include "util/version.h"
 
+QString nameForPreset(const PresetInfo& preset);
+namespace {
+    bool presetInfoNameComparator(const PresetInfo &a, const PresetInfo &b) {
```

Comment:
Our style guide is basically "everything in Google's C++ style guide, plus these exceptions" so that's why :)
https://google.github.io/styleguide/cppguide.html#Namespaces